proto/src/determined/taskconfigpolicy/v1/taskconfigpolicy.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package determined.taskconfigpolicy.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/taskconfigpolicyv1"
+;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// The workload type.
+enum WorkloadType {
```

Comment:
I'm ok with leaving this as is ... we might not need it though. I think postgres will let you define string enums. https://www.postgresql.org/docs/current/datatype-enum.html

Because this is so early in the project, I think I'm leaning towards leaving it in ... and we can remove it if we don't end up needing it.